ec2/sc-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -19,7 +19,7 @@ Metadata:
 Mappings:
   NotebookTypes:
     Rstudio:
-      AMIID: "ami-0ed2999f7cff2e3e3"  # https://github.com/Sage-Bionetworks-IT/packer-rstudio/tree/v1.0.7
+      AMIID: "ami-051310d409a32a5aa"  # https://github.com/Sage-Bionetworks-IT/packer-rstudio/tree/v1.0.7
```

Comment:
@ahayden The AMIID looks right but the comment there should say v2.0.0 no v1.0.7